backend/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:18.19.0-alpine
+
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      freetype-dev \
+      harfbuzz \
+      ca-certificates \
+      ttf-freefont
+
+# ROOT
+
+WORKDIR /app
+
+COPY /*.json .
+
+COPY /*.yml .
+
+COPY /shared ./shared
+
+COPY /backend ./backend
+
+RUN npm install
+
+# BACKEND
+
+WORKDIR /app/backend
+
+RUN npm install
+
+RUN npm run build
+
+#RUN npm run migrate:dev
+
+#RUN npm run seed:run
```

Comment:
initialize db in docker-compose:
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/assets/78302126/fbe74727-f894-41e9-a522-85c5ca38f6f2)